### DISCUSSION TITLE
new version can not verify first certificate

### DISCUSSION THREAD

**Discussion by JACKROMER** [community/community#13966] [Copilot Conversations] [answered]
morning the new version can be used, but afternoon, can not use id , i tried to reload and restart the vscode many times,  but still invalid.
errors like this, and i don't know why！ [
[ERROR] [default] [2022-03-31T08:36:41.186Z] GitHub Copilot could not connect to server. Extension activation failed: "unable to verify the first certificate"] 
i thought network error may due this, but my network is ok ,  anybody help.

**Comment by davecheney** [comment, marked as answer]
Is it possible that you employer, internet service provider, or government may be intercepting your SSL traffic?

**Comment by JACKROMER** [comment]
Thanks! i will retry tomorrow, and  use my  home computer  to test again.

**Comment by renatodru** [comment]
The problem is about trust in chain certificate.
The solution is:
1 - install the extension in VSC "win-ca"
2 - change config "Win-ca: inject (Method to provide fetched root certificates to other externsions)" to **append**.
3 - restart VSC.
